models/userDao.js

```diff
@@ -9,12 +9,14 @@ const signUp = async (user) => {
     (email, first_name, last_name, password, shopping_preference, birthday)
     VALUES (?, ?, ?, ?, ?, ?);`
 
-    await database.query(rawQuery, [user.email,
-    user.first_name,
-    user.last_name,
-    user.password,
-    user.shopping_preference,
-    user.birthday])
+    await database.query(rawQuery, [
+      user.email,
+      user.first_name,
+      user.last_name,
+      user.password,
+      user.shopping_preference,
+      user.birthday,
+    ])
```

Comment:
해당 부분에서 `?`로 매핑하는 코드에 대해 구조분해 할당으로 `user.`이 반복되는 것을 줄여도 좋을 것 같습니다!

그리고 지금 class로 User객체를 하나 생성해주었는데, 생성자 내부에서 초기화하고 있는 각 변수를 camelCase로 수정하는 코드도 반영해주면 좋을 것 같습니다. 

```suggestion
    const { email, firstName, lastName, password, shoppingPreference, birthday } = user;
     
    await database.query(rawQuery, [
      email,
      firsName,
      lastName,
      password,
      shoppingPreference,
      birthday,
    ])
```

Reply:
반영했습니다